skll/config.py

```diff
@@ -474,6 +474,26 @@ def _parse_config_file(config_path):
     # using the defaults?
     do_grid_search = config.getboolean("Tuning", "grid_search")
 
+    # Check `fixed_parameter_list`/`param_grid_list` if `do_grid_search`
+    # is True
+    if do_grid_search:
+        for learner, fixed_params, params in zip(fixed_parameter_list,
+                                                 param_grid_list,
+                                                 learners):
+            overlap_params = set(fixed_params).intersection(set(params))
+            if overlap_params:
+                logger.warning('Both "fixed_parameters" and "param_grids" were'
+                               'specified or the default values were used for '
+                               '{}, which resulted in conflicts between '
+                               'certain parameter(s): {}. Values passed in via'
+                               '"param_grids" will take precedence in these '
+                               'cases.'.format(learner,
+                                               ', '.join(overlap_params)))
+    else:
+        if param_grid_list or fixed_parameter_list:
```

Comment:
What's the problem with specifying `fixed_parameter_list` even if we aren't doing grid search?